protocol/swap/types.go

```diff
@@ -37,7 +37,9 @@ type Info struct {
 	// MoneroStartHeight is the Monero block number when the swap begins.
 	MoneroStartHeight uint64            `json:"moneroStartHeight"`
 	StartTime         time.Time         `json:"startTime"`
-	EndTime           time.Time         `json:"endTime"`
+	EndTime           *time.Time        `json:"endTime,omitempty"`
+	Timeout0          *time.Time        `json:"timeout0,omitempty"`
+	Timeout1          *time.Time        `json:"timeout1,omitempty"`
```

Comment:
Reader's are going to look at these 4 values together and assume that they are connected.  They'll incorrectly assume that `StartTime` is the block timestamp of the `newSwap` transaction and `EndTime` is the block timestamp of the `claim` transaction.  Can we add comments to clarify what each of the 4 timestamps represent?